packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaForm.tsx

```diff
@@ -54,23 +59,34 @@ export const HIERARCHY_END_COMPONENT = (
     })}
   />
 )
+
+export type SubmitHandle = {
+  // Allow the parent component to trigger a submit of the form, so this may be embedded in an arbitrary modal.
+  submit: () => void
+}
```

Comment:
This imperative handle function will let us insert this into any component (an editor modal, or a standalone wizard) and trigger submit programmatically. This is handy because the button that will trigger submit/save probably will be rendered separately from the form (e.g. the modal will render its own save button)